java/ext/javadoctest.py

```diff
@@ -0,0 +1,57 @@
+import subprocess
```

Comment:
Overall, this file should probably be formatted according to Arrow Python guidelines. https://arrow.apache.org/docs/dev/developers/python.html#coding-style

Though that tool won't quite work here, I think, but we just use flake8: https://github.com/apache/arrow/blob/9938f708a718a1ad735a15d6ce2f61d7c5185365/dev/archery/archery/utils/lint.py#L216

Reply:
Nice, let me add that library

Reply:
Added